qa/L0_request_cancellation/scheduler_test.py

```diff
@@ -84,6 +85,20 @@ def _assert_streaming_response_is_cancelled(self, response):
                     cancelled_count += 1
         self.assertEqual(cancelled_count, 1)
 
+    def _get_metrics(self):
+        metrics_url = "http://localhost:8002/metrics"
+        r = requests.get(metrics_url)
+        r.raise_for_status()
+        return r.text
+
+    def _assert_cancel_metrics_sequence_oldest(self, model_name, count, metrics):
+        expected_metric = f'nv_inference_request_failure{{model="{model_name}",reason="CANCELED",version="1"}} {count}'
+        self.assertIn(expected_metric, metrics)
+
+    def _assert_cancel_metrics_sequence_direct(self, model_name, count, metrics):
+        expected_metric = f'nv_inference_request_failure{{model="{model_name}",reason="CANCELED",version="1"}} {count}'
+        self.assertIn(expected_metric, metrics)
```

Comment:
This can just be one helper function called with the two different model names, right?